lib/src/editor/ui/doc_component.dart

```diff
@@ -1,21 +1,19 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_feather_icons/flutter_feather_icons.dart';
-import 'package:flutter_highlight/flutter_highlight.dart';
-import 'package:flutter_highlight/themes/atom-one-dark.dart';
-import 'package:flutterbook/src/editor/ui/copy_text.dart';
+import 'package:flutterbook/src/editor/ui/doc_markdown.dart';
 import 'package:flutterbook/src/editor/ui/styled_icon_button.dart';
 import 'package:flutterbook/src/utils/extensions.dart';
 import 'package:flutterbook/src/utils/radii.dart';
 
 class DocPanel extends StatefulWidget {
```

Comment:
Was it this widget or `doc_markdown` you said no longer needed to be a `StatefulWidget`?

Reply:
was the doc_markdown